.travis.yml

```diff
@@ -8,9 +8,12 @@ sudo: required
 before_install:
   - sudo apt-get install libblas-dev liblapack-dev gfortran -y
   - pip install codecov
+  - pip install flake8
 # command to install dependencies
 install: "pip install -r requirements.txt"
 # command to run tests
-script: nosetests --with-coverage --cover-inclusive
+script: 
+  -  flake8 .
```

Comment:
I think it would be better to run flake8 under before_script flag, as there is no reason to run all the tests if the code fails PEP8 compliance. What do you think? Thanks.